src/Language/SystemVerilog/AST/Expr.hs

```diff
@@ -138,6 +138,8 @@ instance Show Expr where
         showString " : " .
         shows f .
         showChar ')'
+    showsPrec _ (Call (Ident e) (Args [] [])) | "$" `isPrefixOf` e =
+        shows (Ident e)
```

Comment:
Is this purely a stylistic change?

Reply:
Actually, icarus was getting mad at `$time()` vs `$time`. Icarus seems to be accurate with the LRM:

From 3.7.3 and A.8.2
```
system_function_call ::= system_function_identifier
    [ ( expression { , expression } ) ]

system_task_enable ::=
    system_task_identifier [ ( [ expression ] { , [ expression ] } ) ] ;
function_call ::= hierarchical_function_identifier{ attribute_instance }
    ( expression { , expression } )
```

Reply:
I'm not sure I understand. `system_function_call` has square brackets around the argument list including the parens, meaning it should be optional. What am I missing?

Reply:
Verilog has square brackets around the parentheses, but not around the first expression. So if parentheses exist, they must have at least 1 expression